### DISCUSSION TITLE
Bash Support

### DISCUSSION THREAD

**Discussion by trymeouteh** [biomejs/biome#4733] [Ideas]
To have an official support for formatting bash scripts.

**Comment by arendjr** [comment]
Heh, I guess it would be a fun one to have, but given that we target the web ecosystem I don’t think we’ll prioritize this soon :)

**Comment by trymeouteh** [comment]
I found a formatter for Bash that is written in Go which performs very well!

shfmt
https://github.com/mvdan/sh

Visual Studio Code & VSCodium Extension
https://github.com/foxundermoon/vs-shell-format
